### ISSUE TITLE
Enhancement: add ability to set templates for email messages

### ISSUE THREAD

**Issue by louking** [jwag956/flask-security#233]
E.g., may want to have more information in the reset password instructions. This could be done similar to Template Paths, e.g., SECURITY_LOGIN_USER_TEMPLATE to specify the new template.

P.S. Thanks for picking up development of this useful framework!

**Comment by jwag956** [comment]
Always have been - as documented here:
https://flask-security-too.readthedocs.io/en/stable/customizing.html#emails

This doesn't allow for changing the path - but does easily allow for overriding. Is that sufficient?

**Comment by louking** [comment]
Hmm, will try but for some reason I wasn't able to get the general templating to find my templates/security folder for (e.g.,) login_user.html. I had to copy the file, rename it (to login_user.jinja2), then use the path template configuration to find it.

I am guessing this is because I have an non-standard template finder, starting at https://github.com/louking/runningroutes/blob/f42f768071158b8525aad5c39cbb3ea58802b126/runningroutes/__init__.py#L73 but I didn't spend any time debugging this because I was able to get the path template to work.

I use this technique to share templates and javascript across different files.
